css/properties/background-repeat.json

```diff
@@ -79,6 +79,142 @@
             }
           }
         },
+        "no-repeat": {
+          "__compat": {
+            "support": {
+              "chrome": {
+                "version_added": "1"
+              },
+              "chrome_android": "mirror",
+              "edge": {
+                "version_added": "12"
+              },
+              "firefox": {
+                "version_added": "≤15"
```

Comment:
Why "≤15"? The background-repeat property without the no-repeat, repeat, repeat-x and repeat-y values is not functional... So, this seems very safe to be marked as Firefox 1.

Reply:
I couldn't test Firefox 14 and earlier because the collector's property testing function isn't compatible (an argument issue, I believe).  So, at first, I would add this range (or "≤16" before I realized I could test on Fx 15) just to play it safe.  Eventually, I realized that marking the value as supported since the earliest version was a safe bet.